app/models/miq_schedule.rb

```diff
@@ -217,6 +217,18 @@ def action_check_compliance(obj, _at)
     _log.info("Action [#{name}] has been run for target type: [#{obj.class}] with name: [#{obj.name}]")
   end
 
+  def action_automation_request(_klass, _at)
+    uri_parts = {}.tap do |part|
+      part['class'] = filter[:uri_parts][:class] || "Process"
+      part['namespace'] = filter[:uri_parts][:namespace] || "/System"
```

Comment:
Can we put this automate knowledge outside of MiqSchedule.  I don't think MiqSchedule should know about Process, /System, or keys within filter.  Maybe we a method in `AutomationRequest` to build the parameters from the filter.  

Note, most if not all other schedules use MiqExpression objects in the filter column.  Verify others use hashes before you rely on that.  I do not know if there are blind assumptions that the filter is a MiqExpression object.